fast_api/readme.md

```diff
@@ -0,0 +1,24 @@
+# FastAPI Temporal Workflow Template
+
+This repository provides settings for [FastAPI](https://fastapi.tiangolo.com) and the Temporal Python SDK for durable workflows.
+
+## Getting Started
+
+To use this template, follow these steps.
+
+Start the Worker:
+
+```bash
+python3 run_worker.py
```

Comment:
I think they should use `poetry run python` which ensures they are in the virtual env with the right dependencies.